dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1240,6 +1240,7 @@ def create_user(name, url: '/users.json', code: 201, **user_opts)
 end
 
 And(/^I save the section url$/) do
+  wait_short_until {steps 'Then I should see the section table'}
```

Comment:
Yay! This should presumably help with other tests that load the teacher homepage and use this step too.